docs/src/main/asciidoc/config-your-quarkus-applications-using-a-yaml-file_howto.adoc

```diff
@@ -0,0 +1,407 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="configure-a-yaml-application-properties-file"]
+= Configure a YAML application properties file
+include::_attributes.adoc[]
+:categories: core
+:summary: Use `application.yaml` instead of `application.properties`.
+
+Optionally, you can use a YAML-formatted file, `application.yaml`, instead of the default `application.properties` file to configure your application.
+
+link:https://en.wikipedia.org/wiki/YAML[YAML] is a popular format.
+For example, Kubernetes relies on YAML to write various resource descriptors.
+
+.Prerequisites
+
+* You have installed JDK 11 or later and set the `JAVA_HOME` environment variable to specify the location of the Java SDK.
+
+* You have installed Apache Maven {maven-version}.
```

Comment:
I'm unhappy about that. We have a prerequisites include that we should use for all the guides. If it needs adjustments, I can tweak it but we need to use it everywhere.

Reply:
Thanks for the reminder. I'm new to the community docs. I'll add this information to the [Contribute to Quarkus documentation](https://quarkus.io/guides/doc-contribute-docs-howto) topic: https://github.com/quarkusio/quarkus/issues/33227

Reply:
Updated. Thanks.